cmake/3rdParty/FindPIX.cmake

```diff
@@ -6,15 +6,14 @@
 #
 #
 
-file(TO_CMAKE_PATH "$ENV{ATOM_PIX_PATH}" ATOM_PIX_PATH_CMAKE_FORMATTED)
+if(LY_PIX_ENABLED)
+    file(TO_CMAKE_PATH "${LY_PIX_PATH}" PIX_PATH)
+    message(STATUS "PIX found: ${PIX_PATH}")
 
-if(EXISTS "${ATOM_PIX_PATH_CMAKE_FORMATTED}/include/WinPixEventRuntime/pix3.h")
     ly_add_external_target(
         NAME pix
+        3RDPARTY_ROOT_DIRECTORY "${PIX_PATH}"
         VERSION
-        3RDPARTY_ROOT_DIRECTORY ${ATOM_PIX_PATH_CMAKE_FORMATTED}
         INCLUDE_DIRECTORIES include
     )
 endif()
```

Comment:
What controls LY_PIX_ENABLED?  I'm unclear how pix is optional after this change?

Reply:
Just a cmake variable. Enabling pix requires you to pass `-DLY_PIX_ENABLED=ON` and optionally `-DLY_PIX_PATH` to the WinPixEventRuntime folder.